scripts/services/setConfig.ts

```diff
@@ -1,4 +1,12 @@
 import {
   compat,
+  types as T
 } from "../deps.ts";
-export const setConfig = compat.setConfig;
\ No newline at end of file
+export const setConfig: T.ExpectedExports.setConfig = async (effects, input ) => {
+
+  const depsLnd: T.DependsOn = {lnd: ['synced']}
+
+  return await compat.setConfig(effects,input, {
+    ...depsLnd,
```

Comment:
why destructure? why not pass it directly in?

Reply:
@Blu-J ?

Reply:
so.... does this need to change now? or can I roll with it as is to push LND v15? I don't know how to pass it in directly.

Reply:
@islandbitcoin @dr-bonez @Blu-J is this resolved?

Reply:
Blu-J made the change and I committed it in this PR. closing this thread